vueapp/src/components/SelectMembers.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <v-container fluid fill-height class="d-flex flex-column" style="height: 90%">
+    Select members to add
+    <v-list>
+      <v-list-item v-for="player in store.selectableMembersForWaitingList">
+        <v-container fluid class="d-flex align-center">
+          <v-icon class="mr-2">mdi-account-circle</v-icon>
+          {{ player.name }}
+          <v-spacer></v-spacer>
+          <v-checkbox v-model="selected" :value="player"></v-checkbox>
+        </v-container>
+      </v-list-item>
+    </v-list>
+    <v-btn
+      height="72"
+      min-width="140"
+      prepend-icon="mdi-account-circle"
+      :stacked="true"
+      @click="addSelectedMembers"
+      >
+      Done
+    </v-btn>
+  </v-container>
+</template>
+
+<script setup lang="ts">
+import { Ref, ref } from "vue";
+import { usePlayerStore } from "@/stores/playerStore";
+import { Player } from "../models/player";
+
+const emit = defineEmits(["close"]);
+const store = usePlayerStore();
```

Comment:
Be consistent with the other views
```suggestion
const playerStore = usePlayerStore();
```